src/main/java/org/apache/commons/lang3/RandomStringUtils.java

```diff
@@ -81,6 +81,20 @@ public static String random(final int count) {
     public static String randomAscii(final int count) {
         return random(count, 32, 127, false, false);
     }
+
+    /**
+     * <p>Creates a random string whose length is between the minimum and maximum.</p>
+     *
+     * <p>Characters will be chosen from the set of characters whose
+     * ASCII value is between {@code 32} and {@code 126} (inclusive).</p>
+     *
+     * @param minInclusive the minimum length of the string to generate
+     * @param maxInclusive the maximum length of the string to generate
+     * @return the random string
+     */
+    public static String randomAscii(final int minInclusive, final int maxInclusive  ) {
```

Comment:
I guess the intent would be even clearer if the parameters were called `minLengthInclusive` and `maxLengthInclusive`. What do you think?